Tools/CISupport/ews-build/steps.py

```diff
@@ -2054,7 +2054,9 @@ def run(self):
             yield self._addToLog('stdio', 'Unable to fetch PR number.\n')
             return defer.returnValue(FAILURE)
 
-        query_body = '{repository(owner:"Webkit", name:"WebKit") { pullRequest(number: %s) {timelineItems(itemTypes: LABELED_EVENT, last: 5) {nodes {... on LabeledEvent {actor { login } label { name } createdAt } } } } } }' % pr_number
+        project = self.getProperty('project') or GITHUB_PROJECTS[0]
+        owner, name = project.split('/', 1)
```

Comment:
Do we need to do any error-checking here to ensure that owner, name are not unexpected (e.g.: not from one of the GITHUB_PROJECTS)?

Reply:
@briannafan just added that error checking yesterday, will rebase the PR and you should see it.